content/en/checkout/guide.md

```diff
@@ -238,6 +238,67 @@ It will reply with the capture details:
 
 If you wish to be notified each time a payment is successful, follow our [Notifications documentation](/api/notifications).
 
+### Accepting subscriptions
+
+For periodical payments with a set frequency, you can accept subscription payments in Checkout.
+
+To do so, you will need to specify a payment `type` of `["subscription"]`.
+
+You will also need to define:
+- A `payment.start_time`, that determines when the subscription payments start.
+- A `payment.frequency` that sets the periodicity of the payments (weekly, monthly, etc). Available values are:
+  - `"1D"` (daily)
+  - `"1W"` (weekly)
+  - `"2W"` (biweekly)
+  - `"1M"` (monthly)
+  - `"2M"` (bimonthly)
+  - `"3M"` (trimonthly)
+  - `"4M"` (quadmonthly)
+  - `"6M"` (semiannually)
+  - `"1Y"` (annually)
+  - `"2Y"` (biannually)
+  - `"3Y"` (triannually)
+- Either a `payment.expiration_time` or a `payment.max_captures`, so that the subscription has an end eventually.
+
+Check the [reference](/checkout/reference#checkout) for additional details and all the available properties.
+
+An example request payload would therefore be:
+
+```json
+{
+  "type": ["subscription"],
+  "payment": {
+    "methods": ["cc", "dd"],
+    "type": "sale",
+    "capture": {
+      "descriptive": "Descriptive Example"
```

Comment:
`Example` to lower case?

Reply:
Makes sense.

Reply:
Decided to go with a more interesting example as discussed below.